### DISCUSSION TITLE
权限错误：DMMClient：需要管理员权限

### DISCUSSION THREAD

**Discussion by kari-basi** [NateScarlet/auto-derby#119] [Q&A] [answered]
![](https://cdn.discordapp.com/attachments/443007074432188416/868667868961710170/Screenshot_622.png)

**Comment by NateScarlet** [comment, marked as answer]
用管理员权限运行啊 这错误信息哪里写的不明白吗

如果你不知道怎么用管理员权限运行那就用 launcher.cmd 启动

**Reply by kari-basi** [reply]
抱歉我不懂中文所以看不懂已经准备好的步骤，而且如果你通过launcher.cmd打开它会是这样的
![Screenshot_626](https://user-images.githubusercontent.com/87919684/126900867-91fcffac-f4eb-43c4-b06b-8d54131b14ef.png)

**Reply by kari-basi** [reply]
> Run with administrator privileges, don’t you understand where the error message is written?
> 
> If you don’t know how to run with administrator privileges then use launcher.cmd to start

没有视频教程吗？因为我完全不了解脚本和其他

**Reply by NateScarlet** [reply]
Avoid use space in folder name, This is a bug of lanucher #120
